src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE_ARR = 10;
+    private int size = 0;
```

Comment:
primitive fields of the classes have default values, int has 0, so we can not initialize it
```suggestion
    private int size;
```